testing/features/support/components/disclosure/example.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Disclosure
+  class Example < ::Base
+    set_url "/iframe.html?id=components-disclosure--example&args=&viewMode=story"
+
+    element :toggle, ".cads-disclosure__toggle"
```

Comment:
Not sure if it's better for me to add `data-test-id` instead

Reply:
I don't mind the classes for design system components as class names are ultimately part of the public API. `data-testid` would be fine too.